site/en/tutorials/load_data/tf-records.ipynb

```diff
@@ -496,7 +684,35 @@
       "cell_type": "code",
       "source": [
         "filenames = ['test.tfrecords']\n",
-        "dataset = tf.data.TFRecordDataset(filenames)"
+        "dataset_raw = tf.data.TFRecordDataset(filenames)\n",
+        "\n",
```

Comment:
I would get rid of the empty line here for consistency with `dataset_parsed`

naming nit: I suggest using `raw_dataset` and `parsed_dataset`